meson.build

```diff
@@ -31,6 +31,9 @@ endif
 with_coverage = get_option('with_coverage')
 if with_coverage == 'true'
    add_project_arguments('-coverage', language : 'c')
+    add_project_arguments(
+    '-fprofile-dir=' + join_paths(get_option('prefix'), get_option('localstatedir'), 'tmp', 'bluechi-coverage'), 
```

Comment:
This has probably been discussed before, but I wonder why we need to add the prefix etc. to the path and not simply use `/tmp/bluechi-coverage`? The coverage is for test purposes only, so I think its better to be static here - so we can rely on it later, e.g. when collecting those files. Using options this seems harder.

Reply:
Ok, I am fine with that. Just wanted to ask :)